### DISCUSSION TITLE
What's the best way to contribute to SS14 if you aren't a coder?

### DISCUSSION THREAD

**Discussion by Swirl-droid** [space-wizards/space-station-14#4089] [Questions]
I'm really new to the SS14 and Github community. I'm just wondering what the best ways for someone like me to contribute to this project.

**Comment by PaulRitter** [comment]
Let me try to sum up everything that comes to mind:

First and foremost, **please** ask for support in the `#howdoicode`-channel in our discord. We are happy for every contribution and want you to learn. (most of our maintainers started off at scratch too, believe me) **There are no dumb questions!**

## Adding YAML-Prototypes
YAML is somewhat easier than coding since you are only declearing stuff and not actively coding logic. Let me explain:
YAML-Prototypes are static definitions and are used for virtually any content we have. That includes: orders for the cargo-console, technologies for r&d, items, mobs, etc, you get the idea
To get started with this, read [the yaml guide](https://hackmd.io/xcccOLkzSLqM0qF3g7K5eA).
After reading the yaml-guide, you can try to [create your first item](https://hackmd.io/faoh09kORDqor2wqyzWWLw).

## Writing wiki articles
This goes for basically **anything**. Came to mind since @TimrodDX recently wrote [a neat guide on how to set up the AME](https://hackmd.io/LBdaekAGSxWfTeOZeN6X7A). This of course requires you have played the game.

Last but not least, since i think it should be mentioned anyway:
## Code!
**Dont know C#?** We created a page for that: https://hackmd.io/_8TFYXDLQMm-63DjpmIRDQ
**You know C# but want to learn about Robust?** We have a page for that too: https://hackmd.io/x2JdZV8pS2W7b-pM2bsstA
Additionally, we have a [book hosted at hackmd(very very wip)](https://hackmd.io/ngN752w8RdemV8NQ577Qrg) aimed to help new contributors get their bearings with robusttoolbox (our engine).
